app/controllers/comments_controller.rb

```diff
@@ -1,10 +1,8 @@
 class CommentsController < ApplicationController
 
   def index
-    @comments = @commentable.comments.ordered.page(params[:page]).per(3)
-    respond_to do |format|
-      format.js { render 'account/comments/index' }
-    end
+    @comments = @commentable.comments.includes(:user).ordered.page(params[:page]).per(5)
+    @current_user = current_user
```

Comment:
you don't need this instance variable here, rails has helper `current_user` for current user